packages/bot-web-ui/src/components/flyout-block-group.jsx

```diff
@@ -5,10 +5,13 @@ import { localize } from '@deriv/translations';
 import classNames from 'classnames';
 import FlyoutBlock from './flyout-block.jsx';
 
+const Hr = () => <div className='flyout__hr' />;
+
```

Comment:
we only use it once do we need to define const for Hr?

Reply:
Actually, I wanted it to be separate since it's for creating a line. In that case, I had to write a comment for that. What do you think?

Reply:
> Actually, I wanted it to be separate since it's for creating a line. In that case, I had to write a comment for that. What do you think?

it's only a suggestion 😄 , as we only use it online 33 you can simply refactor line 33 to something like this
{is_variables_set && <div className='flyout__hr' />}

Reply:
Let me not to change it, I like this one more, it's more readable, we already have lots of `div`s there.

Reply:
IMO there is no need to add abstraction like this